### ISSUE TITLE
english translation error

### ISSUE THREAD

**Issue by VaporeonScripts** [MagicHarp/confluence#21]
![image](https://github.com/user-attachments/assets/acff08a0-a939-42fc-8d1c-0fd72c8f0d79)
"have" should be changed with "has" because part of the he/she/it pronoums

**Comment by VaporeonScripts** [comment]
![image](https://github.com/user-attachments/assets/992e5e1a-57c5-4199-9886-5c76fbb0f729)
It is dropped by the Wither.
![image](https://github.com/user-attachments/assets/36a0a81d-ec99-414e-844e-6e2f4e22362e)
(it has*)
![image](https://github.com/user-attachments/assets/2b35335c-a2f3-44df-9052-f60d5c42b001)
same here
![image](https://github.com/user-attachments/assets/85644a00-ae85-4cee-a400-4e57796aed79)
basically, "it have" will need to change to "it has"

**Comment by VaporeonScripts** [comment]
almost every curio got that typo, hope it helps :D

**Comment by VaporeonScripts** [comment]
![image](https://github.com/user-attachments/assets/3e244c75-77b7-4b6a-9963-a50606332206)
The item can be obtained through Archaeology.
![image](https://github.com/user-attachments/assets/96118227-0306-4884-b226-8e7ea5b065a6)
has a chance*

**Comment by VaporeonScripts** [comment]
basically any thing that says "[charm name] have" automatically needs to be changed to "[charm name] has"

**Comment by westernat** [comment]
Thanks you vevry much! Because we developers are not native English speakers, all translations are done by machine.

**Comment by VaporeonScripts** [comment]
I figured that,no problem!!

**Comment by VaporeonScripts** [comment]
is that going to be ported to 1.21.1? (asking cuz i only saw the 1.20.1 update xd )
